modules/core/04-channel/keeper/upgrade.go

```diff
@@ -95,7 +95,7 @@ func (k Keeper) ChanUpgradeTry(
 
 	connection, err := k.GetConnection(ctx, channel.ConnectionHops[0])
 	if err != nil {
-		return types.Upgrade{}, errorsmod.Wrap(err, "failed to retrieve connection using the channel connection hops")
+		return types.Upgrade{}, errorsmod.Wrapf(err, "failed to retrieve connection using the channel connection hops (%s)", channel.ConnectionHops[0])
```

Comment:
I think the idea here was that the `err` already contains the connection identifier.

Reply:
Ah, yeah, true, because we're using the interface function. I will revert this one.